### PR TITLE
Regenerate missed FPs / #422

### PR DESCRIPTION
**Pull request by chschlue**
Regenerate missed FPs in the wake of https://github.com/pointhi/kicad-footprint-generator/pull/422

Plus OnSemi_VCT, current generator generates differently-chamfered pads.

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [ ] Provide a URL to a datasheet for the footprint(s) you are contributing
- [ ] An example screenshot image is very helpful 
- [ ] If there are matching symbol or 3D model pull requests, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required
- [ ] Give a reason behind any intentional library convention rule violation.

---

Be patient, we maintainers are volunteers with limited time and need to check your contribution against the datasheet. You can speed up the process by providing all the necessary information (see above). And you can speed up the process even more by providing a dimensioned drawing of your contribution. A tutorial on how to do that is found here: https://forum.kicad.info/t/how-to-check-footprint-correctness/9279 (This is optional!)

### CODE REVIEW

**Comment by evanshultz** [comment]
@chschlue 
Thanks for taking care of this!

I notice `SO-20-1EP_7.52x12.825mm_P1.27mm_EP6.045x12.09mm_Mask3.56x4.47mm_ThermalVias` has too-small via annular rings, according to our library. Probably the datasheet used shows smaller ones or it just selected arbitrarily. Could you please make any script changes to avoid errors here?

Other than than, everything looks fine and I'll merge.

**Comment by chschlue** [comment]
Good catch. Missed that among the false positives.

**Comment by chschlue** [comment]
@evanshultz Should be good to go now.

**Comment by chschlue** [comment]
Done.

**Comment by chschlue** [comment]
https://github.com/pointhi/kicad-footprint-generator/pull/493

**Comment by evanshultz** [comment]
Why exclude `SO-20-1EP_7.52x12.825mm_P1.27mm_EP6.045x12.09mm_Mask3.56x4.47mm_ThermalVias`? I can only assume it is in the script if it was updated before.

**Comment by chschlue** [comment]
Of course.
It was introduced in https://github.com/pointhi/kicad-footprint-generator/pull/474/commits/7eeecbc606f976edeb2f7630261a1448fabf2bc2 and is not needed until https://github.com/KiCad/kicad-symbols/pull/1733 is merged.
I'm wondering whether this should actually get its MF-specific name (Texas_PowerPad_SO-20...) and if following TI's AN that pedantically is necessary (we don't usually do that and I'm afraid it's not possible without violating KLC).

TL;DR: I'm not touching that abomination today.

**Comment by evanshultz** [comment]
@chschlue 
It's been a while. It looks like this should be OK to merge, right?

**Comment by chschlue** [comment]
I think so.
